xmake.lua

```diff
@@ -46,4 +46,8 @@ on_install(function(target) end)
 
 includes("deps")
 includes("UE4SS")
-includes("UVTD")
\ No newline at end of file
+includes("UVTD")
+
+-- TODO: Remove this before the next release. It only exists to maintain backwards compat
```

Comment:
@Buckminsterfullerene02 Please add this to the release checklist, we don't want to forget this.
Make sure to mark it as to be done after the release, not before.

Reply:
Is there any reason to have this in the first place? There are hardly any developers that are using xmake yet, let alone the dev branch of the template, since it has not been in a release.

Reply:
Since it's not been officially released, the question is simple: Do we want to subject early adopters and testers to a bit of pain with an unknown solution (to them, considering xmake is new to all of us) in order to save a bit of hassle for ourselves, or do we want to take on a slight hassle and provide a solution ?
If this was some huge hassle for us, it would be different, but as far as I can tell, this seems very simple.

Reply:
Okay fine you are right